### PR TITLE
Adding ELI5 video to README

### PR DESCRIPTION
**Pull request by dmitryvinn**
## Summary

Here at Meta Open Source, we've been creating short-form videos to explain Meta OSS projects in the simplest terms possible. We started embedding these videos into project pages as you can see in [Docusaurus home page](https://docusaurus.io/), [Jest](https://jestjs.io/), [Litho](https://fblitho.com/), [Hydra](https://hydra.cc/), etc.

## Test plan

image

### CODE REVIEW

**Comment by msaroufim** [comment]
Hi @dmitryvinn this is timely, I'm making a major refactor to our docs 

I think it would be nice to add the ELI5 near the end in the news section we have. If this good lemme know and we can go ahead and close this PR and merge the video with the new README

https://github.com/pytorch/serve/pull/1424/commits/115ef4becdbcac33748a4ec716fe5a1588b58747

**Comment by dmitryvinn** [comment]
@msaroufim that sounds great, let's do that!
Also, do you have plans to update Pytorch's site itself? I would love to add this link in here too https://pytorch.org/serve/

**Comment by msaroufim** [comment]
Yeah that's a good idea I need to revamp pytorch.org/serve

Let me close this task and open a new one
